lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -29,7 +29,7 @@ def source_dir
 
 def fixtures(category)
   begin
-    fixtures = YAML.load_file(".fixtures.yml")["fixtures"]
+    fixtures = YAML.load_file( ENV['FIXTURES_YML'] || '.fixtures.yml' )['fixtures']
```

Comment:
puppetlabs_spec_helper is scheduled for an overhaul, so even if I merge this, I don't know whether there'll be soon a release of any of this, nor if the future interface will look like this.

Other than that, I suppose this could be merged, and looking at https://github.com/puppetlabs/puppetlabs_spec_helper/pull/106 , a release might be useful anyways, if it is not too much of a hassle. @hunner?